chia/types/spend_bundle_conditions.py

```diff
@@ -3,5 +3,9 @@
 import chia_rs
 
 ELIGIBLE_FOR_DEDUP = chia_rs.ELIGIBLE_FOR_DEDUP
-Spend = chia_rs.Spend
+if hasattr(chia_rs, "SpendConditions"):
+    SpendConditions = chia_rs.SpendConditions
+else:
+    # Fallback to the old name if the new name does not exist
+    SpendConditions = chia_rs.Spend
```

Comment:
I would hold off on this and make this change when we bump `chia_rs` next

Reply:
I'm referring to supporting both old and new `chia_rs`. I think it's fine to only support the version that we depend on right now.

Reply:
the circular dependency between chia_rs and chia-blockchain makes this a bit tricky